cmd/minikube/cmd/start_flags.go

```diff
@@ -101,6 +101,7 @@ const (
 	deleteOnFailure         = "delete-on-failure"
 	forceSystemd            = "force-systemd"
 	kicBaseImage            = "base-image"
+	dockerMounts            = "docker-mounts"
```

Comment:
any chance we use the same flag as the one we use for the VMs ?

Reply:
Do you mean --mount-string option or something else?

Reply:
I am open to any solution that doesn't make it docker driver specifi.
--mount-string could be a solution 
or maybe --mount-option

Reply:
I agree with @afbjorklund statements in the https://github.com/kubernetes/minikube/pull/8159#issuecomment-629603739.
The `--docker-mounts` has different attributes like the options for read-only, rslave, etc. i think adding this logic behind the `--mount-string` won't make the implementation more uniform as the options only applies for the container drivers.